apis/v1beta1/collector_webhook.go

```diff
@@ -77,3 +90,17 @@ func (c CollectorWebhook) validate(r *OpenTelemetryCollector) (admission.Warning
 	}
 	return warnings, nil
 }
+
+func SetupCollectorWebhook(mgr ctrl.Manager, cfg config.Config, reviewer *rbac.Reviewer) error {
+	cvw := &CollectorWebhook{
+		reviewer: reviewer,
+		logger:   mgr.GetLogger().WithValues("handler", "CollectorWebhook"),
```

Comment:
can we add a WithValues for the version it's reconciling?

Reply:
I have pushed change to add version label to the logger